spec/draper/configuration_spec.rb

```diff
@@ -6,20 +6,40 @@ module Draper
       Draper.configure { |config| expect(config).to be Draper }
     end
 
-    it 'defaults default_controller to ApplicationController' do
-      expect(Draper.default_controller).to be ApplicationController
-    end
+    describe '#default_controller' do
+      it 'defaults default_controller to ApplicationController' do
+        expect(Draper.default_controller).to be ApplicationController
+      end
+
+      it 'allows customizing default_controller through configure' do
+        default = Draper.default_controller
+
+        Draper.configure do |config|
+          config.default_controller = CustomController
+        end
+
+        expect(Draper.default_controller).to be CustomController
 
-    it 'allows customizing default_controller through configure' do
-      default = Draper.default_controller
+        Draper.default_controller = default
+      end
+    end
 
-      Draper.configure do |config|
-        config.default_controller = CustomController
+    describe '#default_query_methods_adapter' do
```

Comment:
What you think to use `context` on the different scenarios, something like:
```ruby
describe '#default_query_methods_adapter' do
  subject { Draper.default_query_methods_adapter }

  context 'when there is no custom adapter' do
    it { is_expected.to eq(:active_record) }
  end

  context 'when using a custom adapter' do
    before do
      Draper.configure do |config|
        config.default_query_methods_adapter = :mongo
      end
    end

    it { is_expected.to eq(:mongo) }
  end
end
```

Reply:
Thanks!